tests/RuleSet/RuleSetsTest.php

```diff
@@ -201,7 +208,7 @@ private static function assertPHPUnitVersionIsLargestAllowed(string $setName, st
         $allowedVersionsForRuleset = array_filter(
             $allowedVersionsForFixer,
             static function (string $version) use ($maximumVersionForRuleset): bool {
-                return strcmp($maximumVersionForRuleset, $version) >= 0;
+                return (int) str_replace('.', '', $maximumVersionForRuleset) >= (int) str_replace('.', '', $version);
```

Comment:
When all versions were `\d\.\d` that was fine, but for `10.0` it is not anymore:
```php
<?php
var_dump(strcmp('7.5', '5.4') >= 0);
var_dump(strcmp('8.3', '5.4') >= 0);
var_dump(strcmp('10.0', '5.4') >= 0);
```
```shell
bool(true)
bool(true)
bool(false)
```

Reply:
[`'1.20' >= '11.0'`](https://3v4l.org/K1lja)? 😉 Why not [`version_compare()`](https://3v4l.org/4q7nl)?

Reply:
> [`'1.20' >= '11.0'`](https://3v4l.org/K1lja)? 😉

Yeah, that would fail for all the non-existing PHPUnit's versions 😛 


> Why not [`version_compare()`](https://3v4l.org/4q7nl)?

That's a good question, why there was `strcmp` and not `version_compare`.